anaconda.spec.in

```diff
@@ -122,7 +122,7 @@ Requires: NetworkManager-team
 Requires: kbd
 Requires: chrony
 Requires: systemd
-%if ! 0%{?rhel}
+%if 0%{?rhel} <= 10
```

Comment:
Sorry, I wasn't very clear about what we need in the bz (https://bugzilla.redhat.com/show_bug.cgi?id=2273333#c3). We need to require systemd-resolved on fedora and ELN (ie rhel > 10).
I read ^ as requiring it on fedora and rhel <= 10.